wandb/sdk/data_types/table.py

```diff
@@ -972,12 +975,9 @@ def from_json(cls, json_obj, source_artifact):
     def iterrows(self):
         """Iterate over rows as (ndx, row).
 
-        Yields:
-        ------
-        index : int
-            The index of the row.
-        row : List[any]
-            The data of the row.
+        Args:
```

Comment:
Shouldn't the types be removed here?

Reply:
yeah, i think we already discussed it in a different thread but the idea is that if we already have type annotation in the function signature, no need to have it in the docs.